csrankings-4.csv

```diff
@@ -910,7 +910,7 @@ Josef Kittler,University of Surrey,https://www.surrey.ac.uk/people/josef-kittler
 Josef Küng,JKU Linz,https://www.jku.at/en/faw/about-us/team/josef-kueng,uy1d_7wAAAAJ
 Josef Pauli,University of Duisburg-Essen,http://www.is.uni-due.de/en/chair/mitarbeiter/prof_josef_pauli,NOSCHOLARPAGE
 Josef Scharinger,JKU Linz,https://www.jku.at/en/institute-of-computational-perception/about-us/people/josef-scharinger,DSgdJYwAAAAJ
-Josef Sivic,Ecole Normale Superieure,http://www.di.ens.fr/~josef,NCtKHnQAAAAJ
+Josef Sivic,Czech Technical University,http://www.di.ens.fr/~josef,NCtKHnQAAAAJ

```

Comment:
Split (< 75%) appointments are not allowed. Please clarify.

Reply:
Dear Emery,

since 1/8/2020, Josef Sivic is 100% at Czech Technical University. He is in the process of preparing a website, but hasn't finished it yet (that is why I did not update the website). Bellow is the text he is planning to put on the page:
"
I am a distinguished researcher at the Czech Institute of Informatics, Robotics and Cybernetics at the Czech Technical University where I lead the Intelligent Machine Perception project and the ELLIS Unit Prague. I am on leave from a senior researcher position at INRIA Paris where I keep close ties to the Willow team where I am an external collaborator.  I completed my PhD at the University of Oxford (2006) working with Professor Andrew Zisserman and received my habilitation degree (2014) at Ecole Normale Supérieure in Paris. After my PhD, I did a post-doc in the Computer Science and Artificial Intelligence Laboratory (CSAIL) at MIT working with Prof. William Freeman. My research is in computer vision - a branch of computer science and engineering that aims to extract information from images. I am particularly interested in developing intelligent systems that can automatically understand complex visual inputs and can learn with only minimal supervision. Between 2014-2019 I led the ERC project LEAP.  
"

Does this help to clarify the issue?

Thank you very much!